### ISSUE TITLE
Servo isn't starting properly with --browserhtml

### ISSUE THREAD

**Issue by NiLSPACE** [servo/servo#11608]
When using `--browserhtml` servo simply starts as a white rectangle in the screen with nothing happening. As you can see in the screenshot it's also barely using any CPU or RAM.

These are the parameters I used to start servo: `./mach run --release --browserhtml`
The specs of my machine is:
```
CPU: Intel I7-4700MQ
GPU: Intel HD Graphics 4600 (Though I had the same result with my NVIDIA GeForce GT 740M)
RAM: Samsung M471B5173BH0-YK0 | 8GB, 1600 MHz
OS:  Windows 10 Education x64
```

![servobrowserhtml](https://cloud.githubusercontent.com/assets/1160867/15799333/617fe94c-2a59-11e6-8403-27a45d35a017.PNG)

---

I forgot to mention that it starts just fine when I tell it to open the index.html file from browser.html
`./mach run --release "./.cargo\git\checkouts\browserhtml-624c884ba7f6dc1c\gh-pages\index.html"`

**Comment by jdm** [comment]
I assume this will be fixed by #11543.

**Comment by NiLSPACE** [comment]
It's working now, thanks.
